modules/rostests/winetests/gdi32/dib.c

```diff
@@ -2430,35 +2670,43 @@ static void draw_graphics(HDC hdc, const BITMAPINFO *bmi, BYTE *bits)
     solid_patblt( hdc,  8, 50, 1, 1, RGB( 0xff, 0, 0 ) );
     solid_patblt( hdc, 10, 50, 1, 1, PALETTERGB( 0xff, 0, 0 ) );
 
-    compare_hash(hdc, bmi, bits, "Colors");
+    compare_hash_broken_todo(hdc, bmi, bits, "Colors", is_ddb, is_ddb);
 
```

Comment:
@AmineKhaldi, All of the following tests fail and we have a nested 256 iteration loop inside another 256 iteration loop with two or more failures that have to be printed on each iteration. Until we can get the SetPixel and GetPixel functions working correctly we need to skip these tests because they cause timeouts. Just above the "for" below we need an "#ifndef __REACTOS__" and after its closing brace at line #2712 we need an "#endif". Then we can benefit from some of these tests. These tests work fine in Windows. Thanks.